lib/open_api_spex/cast/string.ex

```diff
@@ -27,14 +45,26 @@ defmodule OpenApiSpex.Cast.String do
     trimmed = String.trim(value)
     length = String.length(trimmed)
 
-    if length < min_length do
-      Cast.error(ctx, {:min_length, length})
+    if length <= min_length do
```

Comment:
Are you sure? If the min length is `1`, and the actual length is `1`, then it's an error? Shouldn't that be valid?